src/settings.js

```diff
@@ -0,0 +1,27 @@
+// need md5
+import SettingsUI from '../src/ui/SettingsUI';
+import UserService from './service/UserService';
+import User from './model/User';
+
+const ui = new SettingsUI(document);
+const userService = new UserService();
+let currUser = JSON.parse(localStorage.getItem('currUser'));
```

Comment:
only saved ID. no need for `JSON.parse`

Reply:
`JSON.parse(null) === null`,
`JSON.parse('1') === 1`,
so don't need to change